sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonController.java

```diff
@@ -192,6 +195,62 @@ private void mergePersons(PersonGrid personGrid, List<PersonIndexDto> personInde
 		List<EventParticipantSelectionDto> eventParticipantsWithSameEvent =
 			FacadeProvider.getEventParticipantFacade().getEventParticipantsWithSameEvent(firstPersonUuid, secondPersonUuid);
 
+		Map<String, Map<String, Integer>> eventParticipantsByEvent = new HashMap<>();
+		Set<String> eventsWithDuplicatePersonEventParticipants = new HashSet<>();
+		eventParticipantsWithSameEvent.forEach(eventparticipant -> {
```

Comment:
not clear what happens here, wrong naming and complicated algorithm for an apparently easy thing: getting unique event uuids from a list of event participants, if I'm correct

this must be simplified